misk/src/main/kotlin/misk/hibernate/ForceUTCTimeZoneService.kt

```diff
@@ -0,0 +1,19 @@
+package misk.hibernate
+
+import com.google.common.util.concurrent.AbstractIdleService
+import java.time.ZoneOffset
+import java.util.TimeZone
+
+/**
+ * This works around a Hibernate issue when loading LocalDate instances from the database when
+ * running on a machine which doesn't have UTC as the system timezone. All production servers have
+ * UTC as the system timezone so this is essentially a no op in those case.
+ */
+class ForceUTCTimeZoneService : AbstractIdleService() {
```

Comment:
add `@Singleton`

Reply:
I needed this change for testing Hibernate health checks (#258) and it seemed simple enough to just add the relevant line `TimeZone.setDefault(TimeZone.getTimeZone(ZoneOffset.UTC))` to the `ClockModule`